app/src/main/java/org/p2p/wallet/App.kt

```diff
@@ -43,16 +51,16 @@ class App : Application(), Configuration.Provider {
 
         setupCrashLoggingService()
 
-        IntercomService.setup(app = this, apiKey = BuildConfig.intercomApiKey, appId = BuildConfig.intercomAppId)
```

Comment:
i would rather save BuildConfig. to make it more explicit that it comes from BuildConfig